Automattic/pocket-casts-ios
# Transcripts add parsing subtitles library

Refs #1848 

This PR adds the parsing library for subtitles, [SwiftSubtitles](https://github.com/dagronf/SwiftSubtitles/tree/main), and uses it to parse the transcript files and show the text after clean up of timestamps.

It also adds detection if an episode has transcripts available and show/hides the transcripts tab in the player accordingly.

## To test

- Start the app
- Ensure that you have the `transcripts` and `newShowNotesEndpoint` FF enabled
- Open an episode with Transcripts ( Cautionary Tales podcast episode have translations)
- Play the episode
- Open the Full Screen Player
- Open the Transcript tab
- See if you can see the Transcript without the timestamps and scroll around.
- Open a podcast episode without transcripts available, for example The Daily 
- Check that the transcripts tab is not show 

## Checklist
